### PR TITLE
fix to work with guillotina 3

### PR DESCRIPTION
**Pull request by vangheem**

### CODE REVIEW

**Comment by bloodbare** [comment]
@vangheem you can't use travis postgres on tests as its impossible to make it bind to the public interface on travis nodes and hive children can't access it from the minikube.

**Review by bloodbare — CHANGES_REQUESTED** [review]
Tests and pg problem on travis should be fixed.

**Comment by vangheem** [comment]
tests should clear now if I can get another review

**Review by ableeb — APPROVED** [review]

**Review by bloodbare — APPROVED** [review]

File: guillotina_hive/tests/test_hive.py [hunk 1]
```diff
@@ -116,6 +116,7 @@
 
         job = await hive.get_task_status(task.name)
         assert isinstance(job, Job)
+        import pdb; pdb.set_trace()
```
Comment (bloodbare) [inline guillotina_hive/tests/test_hive.py]: PDB!

**Review by bloodbare — CHANGES_REQUESTED** [review]

**Pull request merged**